samples/bluetooth/mesh/ble_peripheral_lbs_coex/src/lb_service_handler.c

```diff
@@ -131,18 +131,20 @@ static void lbs_adv_start(void)
 		if (id < 0) {
 			printk("Unable to create a new identity for LBS (err %d)."
 			       " Using the default one.\n", id);
-			adv_params->id = BT_ID_DEFAULT;
+			lbs_conn_id = BT_ID_DEFAULT;
 		} else {
-			adv_params->id = id;
+			lbs_conn_id = id;
 		}
 
-		printk("Created a new identity for LBS: %d\n", adv_params->id);
+		printk("Created a new identity for LBS: %d\n", lbs_conn_id);
 	} else {
-		adv_params->id = BT_ID_DEFAULT + 1;
-		printk("Recovered identity for LBS: %d\n", adv_params->id);
+		lbs_conn_id = BT_ID_DEFAULT + 1;
+		printk("Recovered identity for LBS: %d\n", lbs_conn_id);
 	}
 
-	err = bt_le_ext_adv_create(adv_params, NULL, &adv);
+	adv_params.id = lbs_conn_id;
+
+	err = bt_le_ext_adv_create(&adv_params, NULL, &adv);
```

Comment:
```suggestion
	err = bt_le_ext_adv_create(adv_params, NULL, &adv);
```

Reply:
https://github.com/nrfconnect/sdk-zephyr/blob/main/doc/releases/migration-guide-3.7.rst?plain=1#L804-L810
we need to update id so our case is the second.